connection-cache/src/connection_cache.rs

```diff
@@ -571,8 +572,11 @@ mod tests {
         fn default() -> Self {
             Self {
                 udp_socket: Arc::new(
-                    solana_net_utils::bind_with_any_port(IpAddr::V4(Ipv4Addr::UNSPECIFIED))
-                        .expect("Unable to bind to UDP socket"),
+                    solana_net_utils::bind_with_any_port(
+                        IpAddr::V4(Ipv4Addr::UNSPECIFIED),
+                        SocketConfigBuilder::new().build(),
+                    )
+                    .expect("Unable to bind to UDP socket"),
```

Comment:
importing `solana_net_utils` just for this test. so maybe it makes sense to just have a separate function called: `bind_with_any_port_with_config` and leave `bind_with_any_port` as is